### ISSUE TITLE
in-browser sandbox service: "Postgres.new" from the Supabase team

### ISSUE THREAD

**Issue by ndrean** [dwyl/learn-postgresql#110]
You may find this YT video (very!) interesting.

[<img width="913" alt="Screenshot 2024-08-13 at 19 28 20" src="https://github.com/user-attachments/assets/8e2ca341-7eda-4654-8316-9e00c59ba1ca">](https://www.youtube.com/watch?v=ooWaPVvljlU)

**Read more** in th blog post below: `PGlite` is an embedded portage as WASM of `Postgres` to the browser, with a single user/connection, from <https://electric-sql.com/>". 

[<img width="1018" alt="Screenshot 2024-08-13 at 19 30 13" src="https://github.com/user-attachments/assets/9b4e10f3-de6f-4292-ba7c-39e089f1ecdd">](https://supabase.com/blog/postgres-new)

There are plenty of possible links to add. Just the the Github of `PGLite`:

[<img width="917" alt="Screenshot 2024-08-13 at 19 10 08" src="https://github.com/user-attachments/assets/e3fc21f1-bb86-4e14-ac7e-11c1800b9d14">](https://github.com/electric-sql/pglite)

**Comment by nelsonic** [comment]
Yeah, saw it on ~~Twitrrrrr~~ “X” earlier. Looks very good for accelerating PG adoption. 👌

**Comment by nelsonic** [comment]
ElectricSQL With Elixir: https://youtu.be/Sn_Ig5n2Oc0

**Comment by ndrean** [comment]
Impressing demo. 
1) Postgres.new LLM integration makes it easy to interact with Supabase
2) Supabase itself manages Pubsub nicely, with or without garanteess

This makes me think to not compete against LLM but rather learn how to use them better 

[<img width="584" alt="Screenshot 2024-09-12 at 18 36 26" src="https://github.com/user-attachments/assets/d9fe0be9-b173-4683-bdf9-6f0495af0b75">](https://x.com/goldmanem/status/1833997771663442067)
